src/us02.py

```diff
@@ -0,0 +1,45 @@
+'''
+Author: Grace Miguel
+I pledge my honor that I've abided by the the Stevens Honor Code.
+'''
+import sys
+import os
+import pandas as pd 
+import Project02
+from datetime import datetime
+import copy
+
+#this function is to determine if the marriage date is after the birth date
+def user02(gedcom_file):
```

Comment:
nit: function name should match file name

Reply:
changed name to file name